core/src/main/java/org/apache/accumulo/core/client/admin/NewTableConfiguration.java

```diff
@@ -121,8 +128,22 @@ public NewTableConfiguration setProperties(Map<String,String> prop) {
    */
   public NewTableConfiguration enableSampling(SamplerConfiguration samplerConfiguration) {
     requireNonNull(samplerConfiguration);
-    SamplerConfigurationImpl.checkDisjoint(properties, samplerConfiguration);
-    this.samplerConfiguration = samplerConfiguration;
+    Map<String,String> tmp = new SamplerConfigurationImpl(samplerConfiguration).toTablePropertiesMap();
+    checkDisjoint(properties, tmp, summarizerProps);
+    this.samplerProps = tmp;
```

Comment:
I cleaned this up and simplified it in commit 40ed96f.  That removed the unnecessary checks and hopefully made it a little more clear.